src/media/js/addon/components/addon.js

```diff
@@ -97,6 +109,8 @@ export class Addon extends React.Component {
           <dd>{this.props.version}</dd>
         </dl>
 
+        {this.renderDescription()}
```

Comment:
We're not using the definition list as the root element though, it's part of the div.